tests/fakeapi/hello.py

```diff
@@ -354,3 +354,12 @@ def more_than_one_scope_defined():
 
 def test_args_kwargs(*args, **kwargs):
     return kwargs
+
+def test_param_sanitization(query = None, form = None):
```

Comment:
Could you please remove the spaces around the parameter default values? Let's try follow one code style consistently.